lms/static/js/discovery/app.js

```diff
@@ -1,70 +1,72 @@
 ;(function (define) {
-
-define(['backbone', 'course_discovery_meanings'], function(Backbone, meanings) {
     'use strict';
 
-    return function (Collection, Form, ResultListView, FilterBarView, FacetsBarView, searchQuery) {
-        //facet types configuration - set default display names
-        var facetsTypes = meanings;
+    define(['backbone', 'js/discovery/collection', 'js/discovery/form', 'js/discovery/result_list_view',
```

Comment:
Can we rename the factory files so that they contain "factory" in the name, per our usual convention (end with _factory)?